core/plugins/org.polarsys.capella.core.model.helpers/src/org/polarsys/capella/core/model/utils/saxparser/SaxParserHelper.java

```diff
@@ -59,4 +61,15 @@ public List<Resource> getResources() {
 
     return eObject;
   }
+
+  public static String getLinkIdFromStatus(String statusMessage) {
```

Comment:
This method has nothing to do in SaxParserHelper which is a helper while parsing EObject. Move this to the validation plugin where things like message, id, link, etc make sense.